python/imageutils.py

```diff
@@ -212,6 +228,8 @@ def reconstruct_block_block(self, legend):
                     #    end_z = 0
                     #prev_b_end = (block_y + shape[0], block_z + end_z, block_x + end_x)
                     #print "End position of write: {0}".format(prev_b_end)
+        if self.proxy is None:
+            self.proxy = self.load_image(self.filepath)
```

Comment:
self.load_image() will only work if the shape described in header and shape of data matches. If i did it in the constructor, it would throw an error as all I would have is the header written to file.